common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -311,6 +310,11 @@ function(_) {
                 this.videoEl.prop('controls', true);
             }
 
+            // Set video poster
+            if (this.config.poster) {
```

Comment:
We also need to update doc string at [#L33](https://github.com/edx/edx-platform/pull/15093/files#diff-cf2bc63baf174f919026ca527dd6f9e7R40) to reflect this new attribute `poster` in config.